tests/client_suite.go

```diff
@@ -108,6 +109,7 @@ func (s *ClientFunctionalSuite) SetupSuite() {
 		nexusoperations.Enabled:                                       true,
 		dynamicconfig.OutboundProcessorEnabled:                        true,
 		dynamicconfig.EnableMutableStateTransitionHistory:             true,
+		dynamicconfig.FrontendRefreshNexusIncomingServicesMinWait:     1 * time.Millisecond,
```

Comment:
note that this has no effect because the call is wrong: https://github.com/temporalio/temporal/pull/5739#discussion_r1583849789
you might want to fix it in this pr also

Reply:
@stephanos Yes it is meant to be in this PR, it makes the tests faster by allowing the registry to update more frequently since we create a lot of services.

@dnr Good catch, thanks.